packages/near-lake-framework/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@near-lake/framework",
+  "version": "0.1.0",
+  "description": "JS Library to connect to the NEAR Lake S3 and stream the data",
+  "author": "NEAR Inc <hello@nearprotocol.com>",
+  "keywords": [
+    "nearprotocol",
+    "near-indexer",
+    "near-lake",
+    "framework"
+  ],
+  "license": "(MIT OR Apache-2.0)",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/near/near-lake-framework-js.git"
+  },
+  "homepage": "https://near-indexers.io",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "clean": "rm -rf ./dist",
+    "dev": "tsc -w"
+  },
+  "dependencies": {
+    "@aws-sdk/client-s3": "^3.32.0",
+    "@near-lake/primitives": "^0.1.0"
```

Comment:
Should we change this to `file../near-lake-primitives` now that this will be published in the main package?

Reply:
This is the error I get when I try to run `npm run build` from the root directory: 

```
@near-lake/framework:build: src/types.ts(3,53): error TS2307: Cannot find module '@near-lake/prim
itives' or its corresponding type declarations.
```